projects/extension/src/containers/Options.tsx

```diff
@@ -186,6 +189,8 @@ const Options: React.FunctionComponent = () => {
                     />
                   )
                 })
+              ) : bg?.uiInterface.chains?.length === 0 ? (
```

Comment:
```suggestion
              ) : bg?.uiInterface.smoldotCrashError ? (
```

Please use the `smoldotCrashError` and `onSmoldotCrashErrorChanged` functions, which were specifically designed to report about crashes.